app/register-organization/page.tsx

```diff
@@ -0,0 +1,283 @@
+                  </FormControl>
+                  <SelectContent>
+                    {ORG_SIZE_OPTIONS?.map((orgSizes, index) => (
+                      <SelectItem key={index} value={orgSizes?.value}>
+                        {orgSizes?.label}
+                      </SelectItem>
+                    ))}
+                  </SelectContent>
+                </Select>
+                <FormMessage />
+              </FormItem>
+            )}
+          />
+
+          <div className="flex flex-col gap-6 sm:flex-row">
+            <FormField
+              control={form.control}
+              name="country"
+              render={({ field }) => (
+                <FormItem className="flex flex-1 flex-col">
+                  <FormLabel required>Country</FormLabel>
+
+                  <Popover
+                    open={countryPopover}
+                    onOpenChange={setCountryPopover}
+                  >
+                    <PopoverTrigger asChild>
+                      <Button
+                        variant="popover"
+                        className={`${field?.value === "" ? "text-placeholder" : "text-textColor"}`}
+                        role="combobox"
+                      >
+                        <span className="line-clamp-1 text-left">
+                          {field?.value
+                            ? COUNTRIES?.find(
+                                (country) => country?.value === field?.value,
+                              )?.label
+                            : "select"}
+                        </span>
+                        <ChevronDown
+                          className={`h-4 w-4 shrink-0 text-placeholder ${countryPopover && "rotate-180"} transition-transform duration-200
+                            `}
+                        />
+                      </Button>
+                    </PopoverTrigger>
+                    <PopoverContent className="p-0">
+                      <Command>
+                        <CommandInput placeholder="Search country..." />
+                        <CommandEmpty>No country found.</CommandEmpty>
+                        <CommandGroup>
+                          <CommandList>
+                            {COUNTRIES?.map((country) => (
+                              <CommandItem
+                                key={country?.value}
+                                value={country?.value}
+                                onSelect={() => {
+                                  form.setValue("country", country?.value);
+                                  setCountryPopover(false);
+                                }}
+                              >
+                                <Check
+                                  className={cn(
+                                    "mr-2 h-4 w-4",
+                                    field?.value === country?.value
+                                      ? "opacity-100"
+                                      : "opacity-0",
```

Comment:
field?.value !== country?.value && "hidden",

use this instead

Reply:
@imrankhan-stixor , this for tick mark when we select any items form dropdown, if we use the condition mentioned by you, this will make the text shift to right when icon becomes visible, so it's best to make opacity-0 to avoid layout shift.